src/Nethermind/Nethermind.TxPool/TxPool.DefaultTxPoolComparer.cs

```diff
@@ -0,0 +1,48 @@
+﻿//  Copyright (c) 2018 Demerzel Solutions Limited
+//  This file is part of the Nethermind library.
+// 
+//  The Nethermind library is free software: you can redistribute it and/or modify
+//  it under the terms of the GNU Lesser General Public License as published by
+//  the Free Software Foundation, either version 3 of the License, or
+//  (at your option) any later version.
+// 
+//  The Nethermind library is distributed in the hope that it will be useful,
+//  but WITHOUT ANY WARRANTY; without even the implied warranty of
+//  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+//  GNU Lesser General Public License for more details.
+// 
+//  You should have received a copy of the GNU Lesser General Public License
+//  along with the Nethermind. If not, see <http://www.gnu.org/licenses/>.
+// 
+
+using System;
+using System.Collections.Generic;
+using Nethermind.Core;
+
+namespace Nethermind.TxPool
+{
+    public partial class TxPool
+    {
+        /// <summary>
+        /// Default ordering by <see cref="Transaction.GasPrice"/> desc and then <see cref="Transaction.GasLimit"/> asc
+        /// </summary>
+        public class DefaultTxPoolComparer : IComparer<Transaction>
```

Comment:
rename to GasPriceBasedTxComparer

Reply:
it is not obvious for others what is the default one

Reply:
GasBasedTxComparer